docs/getting-started.md

```diff
@@ -59,54 +58,103 @@ Install with [Chocolatey](https://chocolatey.org/):
 choco install serverless
 ```
 
-## Initial setup
+## Upgrade
+
+If `serverless` was installed via NPM, upgrade it via:
+
+```bash
+npm update -g serverless
+```
+
+If you installed `serverless` as a standalone binary, read the following section instead.
+
+### Standalone binary
+
+On MacOS/Linux, run:
+
+```bash
+serverless upgrade
+```
+
+On Windows, run:
+
+```bash
+choco upgrade serverless
+```
+
+## Getting started
 
 Run the command below and follow the prompts:
 
 ```bash
+# Create a new serverless project
 serverless
+
+# Move into the newly created directory
+cd your-service-name
 ```
 
-The command will guide you to create a new serverless project.
+The `serverless` command will guide you to create a new project, [configure your AWS credentials](https://serverless.com/framework/docs/providers/aws/guide/credentials/), and optionally set up a free [Serverless Dashboard](https://www.serverless.com/monitoring) account to monitor, troubleshoot, and test your new service.
```

Comment:
Are we staying with _Dashboard_ name, or switching to _Console_  naming (I'm now really confused :) ?

Reply:
Console is in private preview, readers of this documentation cannot sign up and use it rightaway.

Reply:
Thanks for clarification :+1